File: KawalCovid/app/src/main/java/com/litziy/kawalcovid/MainActivity.java
package com.litziy.kawalcovid;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;

import com.litziy.kawalcovid.databinding.ActivityMainBinding;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Timer;
import java.util.TimerTask;

public class MainActivity extends AppCompatActivity {


    ActivityMainBinding binding;
    Date date;
    String meTime = "";
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        setUpClock();

        binding.btnInitLogin.setOnClickListener(v -> {
            startActivity(new Intent(this,HomeActivity.class));
        });
    }

    void setUpClock(){
        SimpleDateFormat sdf = new SimpleDateFormat("kk:mm:ss");
        date=new Date();
        meTime = sdf.format(date);

        String displayedDate = DateFormat.getDateInstance(DateFormat.ERA_FIELD).format(new Date());

        new Timer().schedule(new TimerTask() {
            @Override
            public void run() {
                runOnUiThread(() -> {
                    Date date = new Date();
                    meTime = sdf.format(date);
                    binding.textB.setText(meTime);
                });
            }
        }, 0, 1000);//1000 is a Refreshing Time (1second)

        binding.textU.setText(displayedDate);

    }

}

File: Angklung/app/src/main/java/com/fariz/angklung/MainActivity.kt
package com.fariz.angklung

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.fariz.angklung.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    val binding by lazy { ActivityMainBinding.inflate(layoutInflater) }
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)

        binding.btnPlay.setOnClickListener {
            startActivity(Intent(this@MainActivity,AngklungActivity::class.java))
        }

        binding.btnHistory.setOnClickListener {
            startActivity(Intent(this@MainActivity,HistoryActivity::class.java))
        }

    }
}